cmd/storagenode/main.go

```diff
@@ -291,6 +303,49 @@ func cmdDiag(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
+// maps deprecated config values to new values if applicable
+func mapConfigs(log *zap.Logger) {
+	type config struct {
+		new     *string
+		newFlag string
```

Comment:
nit - is "flag" the right word to be using here? Flag usually refers to boolean values, but these config values are strings. Maybe `newConfig` and `oldConfig`?